src/Pages/Category/Category.js

```diff
@@ -1,8 +1,120 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Category.scss';
 
 export class Category extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productList: [],
+      isShow: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/categoryData.json')
+      .then(result => result.json())
+      .then(categoryData => {
+        this.setState({ productList: categoryData.productList });
+      });
+  }
+
+  onClickListSort = () => {
+    this.setState({
+      isShow: !this.state.isShow,
+    });
+  };
+
   render() {
-    return <div></div>;
+    return (
+      <div className="Category">
+        <div className="banner">
+          <div className="bannerControl">
+            <Link to="/" className="bannerArrow prevArrow"></Link>
+            <Link to="/" className="bannerArrow nextArrow"></Link>
```

Comment:
```suggestion
            <Link to="/" className="bannerArrow prevArrow" />
            <Link to="/" className="bannerArrow nextArrow" />
```
- 공통되는 스타일과 달라지는 스타일을 각기 다른 클래스명으로 분리한 부분 좋습니다 👍 
- 셀프 클로징 태그 적용해주세요

Reply:
감사합니다 셀프클로징 적용했습니다 !